mono/metadata/w32handle.c

```diff
@@ -418,6 +421,9 @@ mono_w32handle_ref_core (MonoW32Handle *handle_data)
 static gboolean
 mono_w32handle_unref_core (MonoW32Handle *handle_data)
 {
+	if (!handle_data)
```

Comment:
This cannot transparently fail. Check in the caller that we do not unref a invalid handle.